### ISSUE TITLE
The property 'source-file-path' is not defined correctly in action.yml

### ISSUE THREAD

**Issue by breadgrocery** [wdzeng/firefox-addon#9]
You define `source-code-path` in action.yml, but attempt to retrieve it using `source-file-path` in your code:

```yml
  source-code-path:
    description: Path to the add-on source code.
    required: false
```

```js
let sourceFilePath = core.getInput('source-file-path', { required: false }) || undefined
```

This causes the following warning when passing `source-file-path`:
```
Warning: Unexpected input(s) 'source-file-path', valid inputs are ['addon-guid', 'xpi-path', 'source-code-path', 'self-hosted', 'jwt-issuer', 'jwt-secret']
Run wdzeng/firefox-addon@v1
```

**Comment by wdzeng** [comment]
Oh my god

**Comment by wdzeng** [comment]
Fixed in v1.1.1.
